README.md

```diff
@@ -337,6 +337,21 @@ INVALID_EMAIL,bob
 
 The error returned should be: `{ "E-Mail" => "is invalid" }`
 
+### Complex quoting (Illegal quoting problem)
+
+```ruby
+CSV.parse("1,Glk,Lev \"The Black Spider\" Yashin,USSR")
+  # => CSV::MalformedCSVError: Illegal quoting in line 1.
+```
+
+The problem is that the CSV comma-delimited text should entirely surround by `"`
+not in part or do not contain `"` at all. You can pass `quote_char` option.
+
+```ruby
+CSV.parse("1,Glk,Lev \"The Black Spider\" Yashin,USSR", quote_char: "\x00")
+  # => [ ["1", "Glk", "Lev \"The Black Spider\" Yashin", "USSR"] ]
```

Comment:
I am not sure about what you mean by disable. By default ruby CSV class [have quote_char: '"'](https://github.com/ruby/ruby/blob/trunk/lib/csv.rb#L1022)
We have a client that produce a CSV file with double quotes in arbitrary word or sequence of words (in Russia we do not use single quote a lot). I have exported this file with a this [quick fix](https://github.com/farpostdesign/csv-importer/blob/quick_fix/lib/csv_importer/csv_reader.rb#L17). This site is now have been tested on staging and for now we have no signs of problem with this quote_char solution.